intern.html

```diff
@@ -0,0 +1,34 @@
+---
+layout: default
+title: "About"
+permalink: "team/interns"
+---
+
+<h1 class="title page-title">Our Intern Program</h1>
+<h2 class="title">What is Teach LA's internship program?</h2>
+<div class="what-is-intern-container">
```

Comment:
nit: maybe we can use something we already have? anything is good tho!
```suggestion
<div class="row-desktop row-desktop-ratio-1-2 row-desktop-reverse">
```

Reply:
might need to swap the `p` tag and the `img` tag for this to work :)